modules/azure-oracle-adbs/README.md

```diff
@@ -0,0 +1,57 @@
+| [oci_database_autonomous_database](https://docs.oracle.com/en-us/iaas/tools/terraform-provider-oci/6.2.0/docs/r/database_autonomous_database.html) | oci resource       |
+
+## Inputs
+
+| Name                            | Description                                                                                                                                                                                                                   | Type   | Default           | Required |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------ | ----------------- | :------: |
+| `location`                      | Resource region                                                                                                                                                                                                               |
+| `nw_resource_group`             | Virtual network resource group name                                                                                                                                                                                           | string | n/a               |   yes    |
+| `nw_vnet_name`                  | Virtual network name                                                                                                                                                                                                          | string | n/a               |   yes    |
+| `nw_delegated_subnet_name`      | Oracle delegate subnet name                                                                                                                                                                                                   | string | n/a               |   yes    |
+| `db_resource_group`             | ADBS resource group name                                                                                                                                                                                                      | string | n/a               |   yes    |
+| `db_name`                       | Database name. The name must begin with an alphabetic character and can contain a maximum of 14 alphanumeric characters. Special characters are not permitted                                                                 | string | n/a               |   yes    |
+| `db_admin_password`             | Password must be between 12 and 30 characters long, and must contain at least 1 uppercase, 1 lowercase, and 1 numeric character. It cannot contain the double quote symbol (“) or the username “admin”, regardless of casing. | string | n/a               |   yes    |
```

Comment:
We could attach the official doc about the requirement incase it changes in the future https://docs.oracle.com/en-us/iaas/api/#/en/database/20160918/datatypes/CreateAutonomousDatabaseBase